server/src/scripts/initUserData.ts

```diff
@@ -0,0 +1,32 @@
+import { faker } from "@faker-js/faker";
+import axios from "axios";
+
+const apiUrl = "http://localhost:8000/users";
```

Comment:
Let's create axios instance with this url. Also please move base url in .env file